pkg/selinux/selinux-client.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+define([
+    "base1/cockpit",
+], function(cockpit) {
+
+"use strict";
+var _ = cockpit.gettext;
+
+var client = {};
+
+/* until we have a good dbus interface to get selinux status updates,
+ * we resort to polling
+ */
+
+// how often to check the status [milliseconds]
+var pollingInterval = 10000;
+
+var statusCommand = "sestatus";
```

Comment:
Ow wow, that's the only way to find out about the status? It would be good to be notified about changes so that the UI doesn't lag up to 10 seconds after running `setenforce`.